package.json

```diff
@@ -130,7 +130,7 @@
     "ws": "7.4.6"
   },
   "engines": {
-    "node": ">=12.10.0",
+    "node": ">=14.10.0",
     "npm": ">=6.0.0"
```

Comment:
kinda funny to have both the `volta` config and `engines` - out of curiosity, do you know why we're specifying `engines`? the only context where I've had to set it before was to let Heroku know what node version a project should use

Reply:
`engines` is for the users of Stencil, where the volta config is for folks developing/hacking on the Stencil compiler itself.

`engines` [doc](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#engines) handles everything but the version of npm/node we should be using for running `npm install`, running npm scripts, etc in the repo: 

> You can specify the version of node that your stuff works on (example omitted)
> And, like with dependencies, if you don't specify the version (or if you specify "*" as the version), then any version of node will do.
> You can also use the "engines" field to specify which versions of npm are capable of properly installing your program. For example: [omitted]

It also isn't strictly enforced:

> Unless the user has set the engine-strict config flag, this field is advisory only and will only produce warnings when your package is installed as a dependency.

Volta on the other hand, is strictly enforced when working on Stencil, although it tries to be as transparent/out of the way as possible.